content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Environment variables reference'
+metaTitle: 'Prisma environment variables reference'
+metaDescription: 'This page gives an overview of all environment variables avalaible for use.'
+tocDepth: 2
+---
+
+## Overview
+
+This document describes different environment variables and their use cases.
+
+See [Connection URLs](../database-reference/connection-urls) for a breakdown of each database connector.
+
+### DEBUG
+
+`DEBUG` is used to enable debugging output in the Prisma Client.
+
+Example setting Prisma Client level debugging output:
+
+```terminal
+# enable only `prisma-client`-level debugging output
+export DEBUG="prisma-client"
+```
+
+See [Debugging](../../concepts/components/prisma-client/debugging) for more information.
+
+### NO_COLOR
+
+`NO_COLOR` if [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) will activate the `colorless` setting for error formatting and strip colors from error messages.
+
+See [Formatting via environment variables](../../concepts/components/prisma-client/working-with-prismaclient/error-formatting#formatting-via-environment-variables) for more information.
+
+### NO_PROXY
+
+`NO_PROXY` is a comma-seperated list of hostnames, IP addresses or IP ranges that do not require a proxy.
+
+```terminal
+NO_PROXY=myhostname.com,10.11.12.0/16,172.30.0.0/16
+```
+
+### HTTP_PROXY
+
+`HTTP_PROXY` is set with the hostname or IP address of a proxy server.
+
+```terminal
+HTTP_PROXY=http://proxy.example.com
+```
+
+### HTTPS_PROXY
+
+`HTTPS_PROXY` is set with the hostname or IP address of a proxy server.
+
+```terminal
+HTTPS_PROXY=http://proxy.example.com
+```
+
+### PRISMA_HIDE_UPDATE_MESSAGE
+
+`PRISMA_HIDE_UPDATE_MESSAGE` is used to when checking if a Prisma update is cached. It's a truthy value.
+
+```js
+const shouldHide = process.env.PRISMA_HIDE_UPDATE_MESSAGE
+```
+
+### PRISMA_GENERATE_SKIP_AUTOINSTALL
+
+`PRISMA_GENERATE_SKIP_AUTOINSTALL` can be set to a truthy value to skip the auto-install of `prisma` CLI and `@prisma/client` dependencies (if they are missing) when using the `prisma generate` command.
+
+### SKIP_GENERATE
+
+`SKIP_GENERATE` can be set to a truthy value to skip running `prisma generate` on npm's postinstall lifecycle hook.
+
+### BROWSER
+
+`BROWSER` is for Prisma Studio to force which browser it should be open in, if not set it will open in the default browser. It's also used as a flag when starting the studio from the CLI.
+
+```js
+BROWSER=firefox prisma studio --port 5555
+```
+
+```terminal
+prisma studio --browser firefox
+```
+
+See [Studio](./command-reference/#studio) documentation for more information.
+
+### PRISMA_BINARIES_MIRROR
+
+`PRISMA_BINARIES_MIRROR` can be used to specify a custom CDN endpoint to download the engines binaries for the CLI/Client. The default value is `https://binaries.prisma.sh`.
```

Comment:
Does not have to be a CDN - any random server works.

We are missing documentation (and concept!) how to get all the relevant binaries at that mirror URL though.

Reply:
Hmm, ok. So maybe just change the CDN part and create a new story for documenting the about how to get the relevant binaries